statsmodels/base/wrapper.py

```diff
@@ -96,10 +96,15 @@ def wrapper(self, *args, **kwargs):
             obj = data.wrap_output(func(results, *args, **kwargs), how)
         return obj
 
-    argspec = getargspec(func)
-    formatted = inspect.formatargspec(argspec[0], varargs=argspec[1],
-                                      defaults=argspec[3])
-
+    try:  # Python 3.3+
+        sig = inspect.signature(func)
+        formatted = str(sig)
+    except AttributeError:
+        # TODO: Remove when Python 2.7 is dropped
+        argspec = getargspec(func)
+        formatted = inspect.formatargspec(argspec[0],
+                                          varargs=argspec[1],
+                                          defaults=argspec[3])
     func_name = get_function_name(func)
```

Comment:
OT: I'm pretty sure in all supported versions we can just use `func.__name__` at this point

Reply:
Seems to be the case